examples/golang/components/sqldb-postgres-query/wadm.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: go-sqldb-postgres-query
+  annotations:
+    version: v0.1.0
+    description: |
+      Demo WebAssembly component using the wasmCloud SQLDB Postgres provider via the wasmcloud:postgres WIT interface
+    wasmcloud.dev/authors: wasmCloud team
+    wasmcloud.dev/source-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-quer/wadm.yaml
+    wasmcloud.dev/readme-md-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-quer/README.md
+    wasmcloud.dev/homepage: https://github.com/wasmCloud/wasmCloud/tree/main/examples/golang/components/sqldb-postgres-quer
+    wasmcloud.dev/categories: |
+      database,sqldb,postgres,golang,example
+spec:
+  components:
+    - name: querier
+      type: component
+      properties:
+        image: file://./build/sqldb_postgres_query_s_s.wasm
```

Comment:
Should this be double `_s_s`? Also, probably this should be `local.wadm.yaml`

We should also add this component to the example component checks -- the provider won't work but at least the component should come up

Reply:
Nope shouldn't be double `_s_s`